game/typechart.py

```diff
@@ -2,6 +2,8 @@
 
 
 class Element(Enum):
+    """Possible types that Pokemon can be"""
```

Comment:
Minor: 'a Pokemon'

Reply:
Changed to `Possible types that a Pokemon can be`